src/NLog.Targets.Seq/Levels.cs

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
Are you happy to add the [file header text](https://github.com/datalust/nlog-targets-seq/blob/dev/src/NLog.Targets.Seq/RenderingsLayout.cs#L1-L13) to the top of this file?